runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler.cpp

```diff
@@ -184,11 +184,11 @@ OutOfLineINL_jdk_internal_foreign_abi_ProgrammableInvoker_initCifNativeThunkData
 		cifArgTypesNode->argumentTypes = (void **)argTypes;
 
```

Comment:
likewise with the pool and mem allocations here re:J9MEM_CATEGORY_VM_FFI

Reply:
Agreed and replaced with `J9MEM_CATEGORY_VM_FFI` for the pool and mem allocations.